src/test/java/core/basesyntax/CalculationImplTest.java

```diff
@@ -0,0 +1,123 @@
+package core.basesyntax;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class CalculationImplTest {
+
+    private static CalculationImpl calculation;
```

Comment:
```suggestion
    private static Calculation calculation;
```
a bit later we will learn that it is better to use more abstract reference types

Reply:
fixed